task_DEV_3/task_DEV_3/Program.cs

```diff
@@ -45,5 +29,31 @@ static void Main(string[] args)
                 Console.WriteLine("Array length cannot exceed 5");
             }
         }
+
+        static int[] CreateNumbersArray(int arrayLength)
+        {
+            int[] numbersArray = new int[arrayLength];
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine($"Enter an array element № {i}");
+                numbersArray[i] = Convert.ToInt32(Console.ReadLine());
+            }
+            return numbersArray;
+        }
+
+        static bool CheckNumbersForPrime(int number)
```

Comment:
Why do you name method as CheckNumbersForPrime instead of CheckNumberForPrime?

Reply:
done